packages/mendel-pipeline/src/pipeline.js

```diff
@@ -19,15 +19,16 @@ function MendelPipeline(options) {
 
     const registry = new MendelRegistry(options);
     const transformer = new Transformer(options.transforms, options);
+    const depsResolver = new DepResolver(options);
 
-    // Pipeline steps
-    const initializer = new Initialize({registry, transformer}, options);
-    const watcher = new Watcher({registry, transformer}, options);
-    const reader = new Reader({registry, transformer}, options);
-    const ist = new IST({registry, transformer}, options);
-    const depsResolver = new DepResolver({registry, transformer}, options);
+    const toolset = {registry, transformer, depsResolver};
```

Comment:
You really did it? if deps will be an accessory, why don't deps run inside the IST/GST workers instead? we would save the AST part, right?

Just asking out of curiosity, really.